runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -755,6 +755,20 @@ public void onGraphicChanged(GraphicChanged event)
 			createGameTimer(IMBUEDHEART);
 		}
 
+		if (config.showHomeMinigameTeleports()
+			&& (actor.getGraphic() == HOME_TELEPORT.getGraphicId()
+			|| actor.getGraphic() == MINIGAME_TELEPORT.getGraphicId()))
```

Comment:
This would create the teleport timer too early--when the final graphic starts, not when it ends. This should be changed to behave more like the animation check where it checks for final animation -> idle animation.